README.md

```diff
@@ -56,6 +56,15 @@ matwrite("matfile.mat", Dict(
 ); compress = true)
 ```
 
+To write in MATLAB v4 format:
+
+```julia
+matwrite4("matfile.mat", Dict(
```

Comment:
calling `matwrite4` with the kwarg `version="v4"` seems redundant.

Reply:
Corrected the typo. Should have been `matwrite`, with kwarg `version="v4"`